clang/include/clang/Tooling/DependencyScanning/ModuleDepCollector.h

```diff
@@ -114,6 +114,10 @@ struct ModuleDeps {
   /// Whether this is a "system" module.
   bool IsSystem;
 
+  /// Whether this is a module where it's dependencies resolve within the
+  /// sysroot.
```

Comment:
I think this should say that the external name resolves within the sysroot to be clear if it's the virtual or external name. Also maybe something about why this exists like "this is used to identify modules that are unlikely to change during normal development".